tests/test_coregion.py

```diff
@@ -62,6 +62,7 @@ def _prepare_models():
     kc.lengthscale.trainable = False
     kc.variance.trainable = False  # variance is handles by the coregion kernel
     coreg = gpflow.kernels.Coregion(output_dim=2, rank=1, active_dims=[2])
+    coreg.W.assign(np.zeros((2, 1)))  # zero correlation between outputs
```

Comment:
why do we want W to be zero here?

Reply:
i suppose what i mean is: why is the local minimum at 0 not a problem here?

Reply:
because here we want to compare equivalence between separate independent GPs and a coregion GP that enforces zero correlations (W=0).